Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM registry.suse.com/bci/python:3.10
+
+COPY . /pms
+WORKDIR /pms
+RUN zypper -n in python310-devel gcc gcc-c++ make && pip3.10 install -r requirements.txt && rm -rf /var/cache
```

Comment:
do you know that there is real need for this ? I assume it should not be the case . In pcw we requiring gcc and python-devel because we compiling django and uwsgi . Here we don't have anything like that . So I would *guess* that it not needed . But if you know something else please prove me wrong :)

Reply:
Yes the SQLAlchemy installation requires this